src/DefinitionGenerator.ts

```diff
@@ -144,13 +144,11 @@ export class DefinitionGenerator {
       operationObj.deprecated = true;
     }
 
-    if (operationObj.requestBody) {
+    if (documentationConfig.requestBody) {
```

Comment:
I may have found what has been identified as "not working". I suspect we are all using this in a slightly different fashion.

`getRequestBodiesFromConfig` generates the OpenAPI requestBody out of the YML requestModels. It throws an exception if no requestModels are set. This always-false if statement was added to avoid this exception.

I believe the expectation is that you can define `requestModels` in your yaml without setting `requestBody`. You may have been setting both in your testing? If you just have requestModels defined (no requestbody), the request model does not get converted into the OpenAPI requestBody.

Perhaps this should be `if (documentationConfig.requestModels)`, or a combination of both.

It would be nice if there was a test bed for all this, so we could document expectations.

Reply:
I think you're right. I believe the original fix was unnecessary and might even have complicated matters. The error was actually relevant.

Biggest issue now is: how can we get any of these proposed fixes merged? 0.4.0 has been broken for a while now...